src/pages/hybrid-application-development/index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Why are we adding this page again?

This is the same as https://guide.freecodecamp.org/mobile-app-development